packages/openneuro-app/src/scripts/datalad/dataset/dataset-query.jsx

```diff
@@ -119,12 +119,19 @@ export const DatasetQueryHook = ({ datasetId, draft, history }) => {
     }
   })
 
+  useEffect(() => {
+    if (error) {
+      throw error
+    } else if (data && data.dataset && data.dataset.snapshots.length === 0) {
+      throw Error('This dataset has no associated snapshots.')
+    }
+  }, [error, data])
+
```

Comment:
This one is better since it does have some logic at least, but maybe we can move the boundary closer to where this actually causes issues and still allow some data to be shown?